docs/developer-guide.md

```diff
@@ -37,8 +38,7 @@ If you decide to make changes to this file, the following manual steps are requi
 ## Release process
 
 To release a new plugin version, the following steps must be followed:
-1. Create a new release-tag. Set a meaningful version number and describe what has changed in the new version.
+
+1. Create a new release-tag on github. Set a meaningful version number and describe what has changed in the new version.
```

Comment:
The list was broken because it had no leading blank line. 🙈 Have a look at the current mess in the docs: https://djinni.xlcpp.dev/djinni-intellij-plugin/developer-guide/#release-process